include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -401,10 +408,23 @@ class BlmcJointModules
         }
     }
 
+    /**
+     * @brief Set the polarities of the joints
+     * (see BlmcJointModule::set_joint_polarity)
+     * 
+     * @param should_reverse
+     */
+    void set_joint_polarities(std::array<bool, COUNT> should_reverse)
```

Comment:
as above, should_reverse does not seem like a very clear name to me, could we call it reverse_polarities?